src/style.css

```diff
@@ -13,7 +13,7 @@
 }
 #right-panel{
```

Comment:
## CSS Styling
- I suggest adding a space after each block in CSS
- Otherwise there will be linter errors